test/apps/iodemo/ucx_wrapper.h

```diff
@@ -197,14 +198,31 @@ class UcxContext {
 
     void recv_io_message();
 
+    typedef std::map<uint64_t, UcxConnection*>              conn_map_t;
+    typedef std::vector<std::pair<double, UcxConnection*> > timeout_conn_t;
```

Comment:
move typedef to start of private section

Reply:
done